storage/bucket_test.go

```diff
@@ -43,6 +44,15 @@ func TestBucketAttrsToRawBucket(t *testing.T) {
 		MetaGeneration: 39,
 		Created:        time.Now(),
 		Labels:         map[string]string{"label": "value"},
+		Lifecycle: Lifecycle{Rules: []storage.LifecycleRule{storage.LifecycleRule{
```

Comment:
Thanks for adding this, it's valuable. But what you really need to do is to add to `TestBucketAttrsToUpdateToRawBucket`.

Reply:
Hey Sorry about that was a previous version. I am trying to run the tests following the contrib.md but struggling (appears to keep timing out). Ill finish the tests blind they will need some attention potentially.